### ISSUE TITLE
Typescript project fails to build - missing typings

### ISSUE THREAD

**Issue by j3ddesign** [apollographql/apollo-engine-js#159]
First off, I love the project, thank you!
I'm currently running into a slight issue when using this within a typescript project, where I am forced to add extra dev dependencies to my `express` project to achieve a successful build.
Dependencies needed: `connect @types/koa @types/koa-bodyparser @types/koa-router @types/restify`

Errors shown without manually including:
```
node_modules/apollo-engine/lib/engine.d.ts(3,23): error TS2688: Cannot find type definition file for 'connect'.
node_modules/apollo-engine/lib/engine.d.ts(4,23): error TS2688: Cannot find type definition file for 'koa'.
node_modules/apollo-engine/lib/engine.d.ts(5,23): error TS2688: Cannot find type definition file for 'koa-bodyparser'.
node_modules/apollo-engine/lib/engine.d.ts(6,23): error TS2688: Cannot find type definition file for 'koa-router'.
node_modules/apollo-engine/lib/engine.d.ts(10,38): error TS2307: Cannot find module 'connect'.
node_modules/apollo-engine/lib/engine.d.ts(11,25): error TS2307: Cannot find module 'koa'.
node_modules/apollo-engine/lib/engine.d.ts(12,41): error TS2307: Cannot find module 'restify'.

```

**Comment by glasser** [comment]
Hmm. These are all devDependencies that are only used by our tests. Are you saying that we are publishing a .d.ts file that contains references to test-only things? Huh.

**Comment by glasser** [comment]
Oh no, you're right, these types are in our API.

**Comment by glasser** [comment]
@j3ddesign Can you try to install the prerelease apollo-version@1.0.4-0 and see if that fixes it?

**Comment by j3ddesign** [comment]
@glasser That did indeed resolve the issue

**Comment by glasser** [comment]
OK. I'm not a TS expert and I want to make sure that this

**Comment by glasser** [comment]
Fixed in 1.0.5. https://www.apollographql.com/docs/engine/proxy-release-notes.html
